nilrt_snac/_Configs/_BaseConfig.py

```diff
@@ -0,0 +1,13 @@
+import argparse
```

Comment:
I think modules (including individual files like this) are normally named in the python `lower_case` styling, rather than like the classes they contain.

Reply:
All are renamed.